CHARTER.md

```diff
@@ -0,0 +1,257 @@
+
+TSC members are expected to regularly participate in TSC activities.
+
+The TSC will meet regularly using virtual conferencing tools. The meeting will be directed by the TSC Chairperson(s). Responsibility for directing individual meetings may be delegated by a TSC Chairperson to any other TSC member. Minutes or an appropriate recording will be taken and made available to the community through accessible public postings.
+
+The TSC may, at its discretion, invite any number of non-voting observers to participate in the public portion of TSC discussions and meetings.
+
+A TSC member may be removed by vote from the TSC if, during a 3-month period, all of the following are true:
+
+- They attend fewer than 25% of the regularly scheduled meetings
+- They do not participate in any TSC votes
+- They do not provide any form of excuse or no excuse is known for their absence
+
+## Section 4: Roles & Responsibilities
+<!-- https://github.com/json-schema-org/community/discussions/286#discussioncomment-4391286 -->
+
+The JSON Schema project is jointly governed by a Technical Steering Committee (TSC) which is responsible for high-level guidance of the project.
+
+The TSC has final authority over this project including:
+
+- Technical direction
+- Project governance and process (including this policy)
+- Contribution policy
+- GitHub repository hosting and administration
+- Establishment of and delegation to working groups or teams
+- Mediating technical conflicts
+- Mediating non-technical conflicts (until a formal Code of Conduct Committee is established)
+
+### Section 4.1 Project Operations & Management (optional)
+<!-- https://github.com/json-schema-org/community/discussions/286#discussioncomment-4391290 -->
+
+The TSC and entire technical community will follow any processes as may be specified by the OpenJS Foundation Board relating to the intake and license compliance review of contributions, including the OpenJS Foundation IP Policy.
+
+### Section 4.2: Decision-making and Voting
+<!-- https://github.com/json-schema-org/community/discussions/286#discussioncomment-4391296 -->
+Note: See comments found in raw view...
+
+The TSC follows a formal consensus seeking decision making model.
```

Comment:
Are we considering adding a limit of time for the consensus process?

Reply:
I asked this sort of question here: https://github.com/orgs/json-schema-org/discussions/338
Only two people replied (thank you!), so I'm just going to have to do SOMETHING and see objections during the review process, I guess.

Reply:
@benjagm I have added some time limits around some parts of the consensus process: https://github.com/json-schema-org/community/pull/325/files#diff-91d16dd057bd39f6431e90a1d78e9224eafafcedc611eba0878d77e6fda86601R179

If this looks OK for now, can you approve and resolve this review comment please? =]

Reply:
Looks good to me. Thanks for this change.